index.html

```diff
@@ -4,11 +4,38 @@
     <meta charset="utf-8">
     <meta name=“viewport” content=“width=device-width, initial-scale=1”>
     <title>Portfolio | Lee Broxson - Javascript Developer</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Share+Tech">
     <link rel="stylesheet" href="vendor/styles/normalize.css">
     <link rel="stylesheet" href="vendor/styles/icons.css">
     <link rel="stylesheet" href="styles/base.css">
     <link rel="stylesheet" href="styles/layout.css">
     <link rel="stylesheet" href="styles/modules.css">
+
+    <!--Template for Projects constructor and page -->
+      <script id="project-template" type="text/x-handlebars-template">
+        <article data-name="{{name}}">
+          <h1 class="project-title">{{title}}</h1>
+          <p class="sub-heading">{{subtitle}} - <span class ="days-ago">{{publishStatus}}</span></p>
```

Comment:
Try some more structural indenting to make the html a little easier to read, ex. the <p> could be indented from the <h1>.